src/MessageTrait.php

```diff
@@ -212,5 +212,14 @@ private function assertHeader($header)
         if ($header === '') {
             throw new \InvalidArgumentException('Header name can not be empty.');
```

Comment:
This is not needed anymore given the regex. Please add empty string to the tests cases.